BurrowsWheelerTransform/BurrowsWheelerTransform/BurrowsWheelerTransform.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace BurrowsWheelerTransform;
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+/// <summary>
+/// Класс, реализующий преобразование Барроуза-Уилера.
+/// </summary>
+public static class BurrowsWheeller
+{
+    /// <summary>
+    /// Выполняет преобразование Барроуза-Уилера над входной строкой.
+    /// </summary>
+    /// <param name="input">Входная строка.</param>
+    /// <returns>Кортеж, содержащий преобразованную строку и позицию исходной строки (Индексация с нуля).</returns>
+    public static (string transformed, int position) Transform(string input)
```

Comment:
По стайлгайду элементы кортежей надо с большой буквы именовать
```suggestion
    public static (string Transformed, int Position) Transform(string input)
```